app/javascript/react/components/RideShow.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ReviewTile from "./ReviewTile";
+
+const RideShow = (props) => {
+  const reviewTiles = props.reviews.map((review) => {
+    return (
+      <ReviewTile
+        key={review.id}
+        title={review.title}
+        body={review.body}
+        rating={review.rating}
+        created_at={review.created_at}
+        username={review.user.username}
+      />
+    );
+  });
+
+  return (
+    <div>
+      <h1>{props.name}</h1>
+      <h2>{props.location}</h2>
+      <img src={props.image_url} alt={props.name} className="propsimage" />
+      {reviewTiles}
+    </div>
```

Comment:
ill allow this but divs should always have a classname, and otherwise you should use a fragment if you dont need the class